website/server/models/user/schema.js

```diff
@@ -73,7 +73,8 @@ let schema = new Schema({
   // We want to know *every* time an object updates. Mongoose uses __v to designate when an object contains arrays which
   // have been updated (http://goo.gl/gQLz41), but we want *every* update
   _v: { $type: Number, default: 0 },
-  migration: String,
+  migration: String, // deprecated
+  migrations: Array, // {name: '20190801_take_this', processed: new Date(), item: 'shield_special_takeThis'}
```

Comment:
can we specify this is an array of mixed objects? or if it always follow the `{name: '20190801_take_this', processed: new Date(), item: 'shield_special_takeThis'}` structure maybe define it?